src/rgw/rgw_op.cc

```diff
@@ -6131,17 +6131,16 @@ void RGWSetRequestPayment::pre_exec()
 void RGWSetRequestPayment::execute(optional_yield y)
 {
 
+  op_ret = get_params(y);
+  if (op_ret < 0)
+    return;
+  
```

Comment:
ok, so we rely on `RGWSetRequestPayment_ObjStore_S3::get_params()` to read the `in_data` that we're forwarding with `forward_request_to_master()`